stock_picking_product_kit_helper/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Kitti U. - Ecosoft <kittiu@ecosoft.co.th>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Stock Picking Product Kit Helper',
+    'summary': 'Set quanity in picking line based on product kit quantity',
+    'version': '12.0.1.0.0',
+    'category': 'Stock',
+    'website': 'https://github.com/OCA/stock-logistics-workflow',
```

Comment:
@kittiu Don't forget to adapt manifest to this repo

Reply:
Fixed